Source/animation/AnimationImpl.swift

```diff
@@ -56,7 +61,12 @@ class BasicAnimation: Animation {
     }
 
     override open func easing(_ easing: Easing) -> Animation {
-        self.easing = easing
+        switch easing {
+        case .spring:
+            spring = easing
```

Comment:
Looks like `easing(.spring)` will result to spring + easy instead of spring + linear.

Reply:
it is not the place which decides that, TimingFunction.swift is

Reply:
Ok, just was curios.